hirundo/dataset_optimization.py

```diff
@@ -150,6 +155,13 @@ class OptimizationDataset(BaseModel):
     def validate_dataset(self):
         if self.dataset_storage is None and self.storage_integration_id is None:
             raise ValueError("No dataset storage has been provided")
+        if self.labelling_type == LabellingType.SpeechToText and self.language is None:
```

Comment:
Enum conventions in python are as in [this link](https://docs.python.org/3/howto/enum.html) as far as I'm aware. Though it is only matter of convention, interesting to see what you think about this.

Reply:
(Capital letters for the values, LabellingType is fine)

Reply:
You use it correctly below RunStatus.PENDING

Reply:
Interesting... You're the first to mention that :) I will change it

Reply:
Done